assets/css/editor.scss

```diff
@@ -45,6 +45,18 @@
 	}
 }
 
+svg.wc-block-editor-components-block-icon {
+	color: rgb(127, 84, 179);
+}
+
+.block-editor-list-view-leaf.is-selected {
+	.block-editor-list-view-block-contents {
+		svg.wc-block-editor-components-block-icon {
+			color: currentColor;
+		}
+	}
+}
```

Comment:
Nit: this CSS selector can be simplified:
* `.block-editor-list-view-block-contents` isn't really needed.
* `svg` isn't needed either because now that we use our own class, we know it's always applied to an svg.

```CSS
.block-editor-list-view-leaf.is-selected .wc-block-editor-components-block-icon {
	color: currentColor;
}
```

Reply:
I really appreciate `nit` comments! You're absolute correct in pointing them out, and I applied the suggested changes right away. I'm now going to merge this PR so that everyone can benefit from these changes. 

Thanks again for reviewing this PR so quickly, @Aljullu! 🙏